app/src/main/java/net/aiscope/gdd_app/ui/mask/customview/MaskLayer.kt

```diff
@@ -204,22 +204,12 @@ class MaskLayer(
     }
 
     fun drawMove(x: Float, y: Float) {
-        currentPath?.run {
-            val (latestX, latestY) = latestPoint
-            val dX = abs(x - latestX)
-            val dY = abs(y - latestY)
-            val touchTolerance = ViewConfiguration.get(context).scaledTouchSlop / currentScale
-            if (dX >= touchTolerance || dY >= touchTolerance) {
-                quadTo(x, y)
-            }
-        }
+        currentPath?.quadTo(x, y)
     }
 
     fun drawEnd() {
         currentPath?.run {
-            val visibleChange: Boolean =
-                if (isCurrentModeDraw()) hasMultiplePoints()
-                else !latestChangeBitmap.sameAs(currentStateBitmap)
+            val visibleChange: Boolean = !latestChangeBitmap.sameAs(currentStateBitmap)
```

Comment:
Since we only use `visibleChange` in the if on the next line, I would prefer to simply have it in the if statement. Also, since we dropped the 'multiplepoints' requirement, I don't think `visible` is a useful name here

Reply:
Done in 4dc9120.